### PR TITLE
load test tweaks

### PR DESCRIPTION
**Pull request by oschaaf**
A couple of things I needed to tweak when trying to run the load tests

### CODE REVIEW

File: devel/mps_load_test.sh [hunk 1]
```diff
@@ -215,11 +215,11 @@ make -j8 CONF=$compile_mode apache_trace_stress_test_server \
 
 # If a custom .so got specified, install it.
 if [[ -n "$custom_so" ]]; then
-  install -c $custom_so /usr/local/apache2/modules/mod_pagespeed.so
+  install -c $custom_so $HOME/apache2/modules/mod_pagespeed.so
```
Comment (jmarantz) [inline devel/mps_load_test.sh]: Is $HOME always right?  For that to go /home/jmarantz/apache2/...

maybe better to have APACHE_DIR which defaults to /usr/local/apache2 if not already defined as an env var.

Comment (oschaaf) [reply devel/mps_load_test.sh]: Well, the release process doc goes:

```
Please load-test using multiple Apache version & MPMs 
..... 
These can all be built using install/build_development_apache.sh, which builds a new ~/apache2
```

So it surprised me that `devel/mps_load_test.sh` pointed to a location that didn't exist on my machine. If we only load test in the release flow, this change works..

Comment (jmarantz) [reply devel/mps_load_test.sh]: Sorry -- we already have the variable I was talking about.  Use $APACHE_DEBUG_ROOT here, defined above as I described.

Comment (oschaaf) [reply devel/mps_load_test.sh]: Nice, done in a971e89cc4c375ab7b00b2be16684af65d2cc941

**Review by jmarantz — APPROVED** [review]

**Review by jmarantz — APPROVED** [review]
You could while you are in here also quote the other usage of APACHE_DEBUG_ROOT but it's up to you.

**Review by jmarantz — APPROVED** [review]

**Pull request merged**